src/Altinn.ExampleClients/PrefillClient/PrefillClient.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
```

Comment:
Is it about time we retire this prefill client? Does it even work any longer?

Reply:
I don't think it works. We move much of the logic it depends on out of Storage and into the App.

Reply:
Official retired.